examples/stable_diffusion_xl/scripts/run_distribute_vanilla_ft_multi_graph_910b.sh

```
@@ -14,6 +14,7 @@ END_DEVICE=$3
 RANK_SIZE=$4
 DATASET_PATH=$5
 
+export HCCL_CONNECT_TIMEOUT=7200
```

Be cautious for over-large timeout. Consider setting HCCL_EXEC_TIMEOUT as well to avoid hiding the error info

---

Running errors will not be hidden due to timeout